code/modules/clothing/masks/gasmask.dm

```diff
@@ -20,6 +20,24 @@
 		"Plasmaman" = 'icons/mob/species/plasmaman/mask.dmi'
 		)
 
+/obj/item/clothing/mask/gas/wirecutter_act(mob/living/user, obj/item/I)
+	if(type != /obj/item/clothing/mask/gas)
+		return
+	if(icon_state == "rebreather")
+		return
```

Comment:
Feel like you should track whether its a rebreather or not using something other than icon state.